pyproject.toml

```diff
@@ -11,9 +12,18 @@ include = ["whylabs_toolkit/monitor/schema/schema.json"]
 [tool.poetry.dependencies]
 python = "^3.8"
 whylabs-client = "^0.6.0"
-pydantic = "^1.10.4"
+pydantic = "1.10.15"
 whylogs = "^1.1.26"
 jsonschema = "^4.17.3"
+typing-extensions = "^4.11.0"
+urllib3 = ">=2.0.2,<2.1.0"
```

Comment:
this could also go as extra dep for diagnoser only

Reply:
It could as long as we dont think the urllib3 2.0.1 problem will impact other uses of the toolkit - I'd worry about large monitor configs, and if its being used in the container. The truncation is really quite a serious issue.

Reply:
it used to be a whylogs-container dependency, but I just checked `master` and it's not, so maybe it's not really an issue

Reply:
I'm assuming I can leave as is?